app/src/test/java/com/example/parkingproject/mvp/presenter/ReleasePresenterTest.java

```diff
@@ -0,0 +1,185 @@
+package com.example.parkingproject.mvp.presenter;
+
+import com.example.parkingproject.entities.Parking;
+import com.example.parkingproject.entities.Reservation;
+import com.example.parkingproject.exceptions.ReleaseMoreThanOneMatchException;
+import com.example.parkingproject.exceptions.ReleaseNoMatchException;
+import com.example.parkingproject.exceptions.ReservationListEmptyException;
+import com.example.parkingproject.mvp.model.ReleaseModel;
+import com.example.parkingproject.mvp.view.ReleaseView;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Date;
+
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class ReleasePresenterTest {
+    private ReleasePresenter presenter;
+    private ReleaseModel model;
+    private ReleaseView view;
+
+    @Before
+    public void setup() {
+        model = mock(ReleaseModel.class);
+        view = mock(ReleaseView.class);
+        presenter = new ReleasePresenter(model, view);
+    }
+
+    //region onReleaseParkingPressed tests
+    @Test
+    public void onReleaseParkingPressed_success() throws ReleaseMoreThanOneMatchException, ReleaseNoMatchException, ReservationListEmptyException {
```

Comment:
I think you can remove the throws part.

throws ReleaseMoreThanOneMatchException, ReleaseNoMatchException, ReservationListEmptyException